bundles/io/org.eclipse.smarthome.io.console/src/main/java/org/eclipse/smarthome/io/console/extensions/GenericConsoleCommandExtension.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.io.console.extensions;
+
+import org.eclipse.smarthome.io.console.Console;
+
+/**
+ * A base class that should be used by console command extension for better inclusion.
+ *
+ * @author Markus Rathgeb - Initial contribution and API
+ *
+ */
+public abstract class GenericConsoleCommandExtension implements ConsoleCommandExtension {
```

Comment:
One comment from my site: The name GenericConsoleCommandExtension is a little bit confusing to me. It looks like an abstract base class for concrete console command extensions (and you also documented it like this in the class java doc). "Generic" is used in other ESH bundles for concrete service implementations (like the GenericItemProvider). We typically use Base or Abstract as prefix for abstract base classes. So "AbstractConsoleCommandExtension" would be a better name.